aten/src/ATen/function_wrapper.py

```diff
@@ -644,7 +644,8 @@ def native_decl():
         # generate the at::native function declarations (i.e. what the user will implement)
         if isinstance(dispatch, dict):
             generated_native_functions = []
-            for _, value in dispatch.items():
+            for key in sorted(dispatch.keys()):
```

Comment:
Sorted should still work on items, provided tuples implement lexicographical comparison (I think they do)

Reply:
Ofc you don’t need to change the code. It’s more of a general note